django/db/models/sql/query.py

```diff
@@ -1099,8 +1092,8 @@ def try_transform(self, lhs, name, rest_of_lookups):
                 "permitted." %
                 (name, lhs.output_field.__class__.__name__))
 
-    def build_filter(self, filter_expr, branch_negated=False, current_negated=False,
-                     can_reuse=None, connector=AND, allow_joins=True, split_subq=True):
+    def build_filter(self, filter_expr, current_negated=False,
```

Comment:
I'll edit this docstring to remove the references to the removed parameers, but please check my edits.